CONTRIBUTING.md

```diff
@@ -1,47 +1,174 @@
-5. Pushes their changes to their fork and raises a Pull Request
-    6. Adds a descriptive title for the PR
-    7. Adds relevant details in the Pull Request description:
-        * Details of any problems that the changes aim to fix
-        * Details of the solution proposed in the Pull Request
-        * Image(s)/gif(s) showing the before and after of functionality/look of the changes made
-        * Any links to issues, code examples, and items of help that may be relevant
-        * Please feel free to use [Markdown](https://help.github.com/articles/basic-writing-and-formatting-syntax/) in your Pull Request
-8. The Owning Team reviews the Pull Request according to the following criteria:
-    * Can the changes be run and tested?
-    * Do the changes clearly indicate what they are trying to achieve?
-    * Are the changes aligned with our coding standards and our architectural approach?
-    * Are the changes compatible with other in-flight changes to the Repository?
-    * Do the changes introduce any new operational concerns e.g. performance, security?        
-9. The Owning Team accepts or rejects the Pull Request with a comment
-    * If rejected, the Requestor can ask the Owning Team for feedback
-    * If accepted, the Owning Team merges the changes into the Repository and creates a new release
-
-Once the Pull Request has been merged, it is the joint responsibility of the Requestor and the Owning Team to:
-   * discuss the optimal timing of pre-production and production releases
-   * validate the functional and operational impact once the release reaches pre-production and production
-
-It is the responsibility of the Requestor's Team alone to deploy/schedule the release into the relevant pre-production and production environment(s). 
+# Contribution Guidelines for assets-frontend
+
+Hello! Thank you for taking the time to contribute to the [HMRC Multichannel Digital Tax Platform](https://hmrc.github.io) (MDTP)
+
+We welcome contributions to assets-frontend. By following these guidelines, you will greatly reduce the amount of feedback (and therefore time) between opening your Pull Request and us merging it.
+
+## How to Help?
+
+We’re always looking for feedback, and help is always welcome. These are the areas where you can help:
+
+* Bug fixes and improvements
+* Proposing new features
+* More tests and better coverage
+* Documentation
+
+## Communication
+
+**Before** starting on a piece of work, **please get in touch**. 
+
+We may already have what you’re planning to do on our backlog, or already be working on it.
+
+Communicating early can also help with design decisions and direction for your approach, increasing the chance of us accepting your work.
+
+* [Raise an issue](https://github.com/hmrc/assets-frontend/issues/new)
+* [Slack channel for getting in touch with frontend developers](https://hmrcdigital.slack.com/archives/community-frontend)
```

Comment:
How would someone know about it otherwise? There's no harm in it being there. If someone doesn't have the right creds then they won't have access.

Perhaps mentioning that membership is required would do?